tests/wpt/metadata/fetch/content-length/parsing.window.js.ini

```diff
@@ -42,16 +42,16 @@
     expected: FAIL
 
   [Input: "Content-Length: 42,42". Expected: 42.]
-    expected: FAIL
+    expected: PASS
```

Comment:
Comma separated values are acceptable as header values

Reply:
Passing test results can be removed from this file.